### PR TITLE
CONSOLE-3389: Use Cypress mock responses for multicluster cluster switcher and pod test

### PR DESCRIPTION
**Pull request by jcaianirh**
Story: https://issues.redhat.com/browse/CONSOLE-3389

### CODE REVIEW

File: frontend/packages/integration-tests-cypress/tests/app/multi-cluster.spec.ts
Comment (spadgett) [inline frontend/packages/integration-tests-cypress/tests/app/multi-cluster.spec.ts:55]: I would also make sure the requests all contain the correct cluster name in the request headers.

File: frontend/packages/integration-tests-cypress/tests/app/multi-cluster.spec.ts
Comment (spadgett) [inline frontend/packages/integration-tests-cypress/tests/app/multi-cluster.spec.ts:17]: Worth a comment here explaining that we switch perspectives to force a re-render.

**Review by spadgett — COMMENTED** [review]
Great start, good to see some coverage of this in CI.

When we enable metrics for spoke clusters, we should look at updating this to mock / check that the pod metrics update as well.

**Review by TheRealJon — COMMENTED** [review]
Just a few nits. This is going to conflict with Steves PR to add a filter to the cluster dropdown, but I think it'll be an easy fix.

**Review by spadgett — APPROVED** [review]
/lgtm

👍

**Comment by jcaianirh** [comment]
QE Approver:
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @RickJWagner

**Comment by spadgett** [comment]
This only adds automated tests, so shouldn't need approvals. @yapei @opayne1 @RickJWagner let me know if you disagree.

/label px-approved
/label docs-approved
/label qe-approved

**Pull request merged**
